examples/run_client_main.cc

```diff
@@ -0,0 +1,30 @@
+#include "src/client/gfs_client.h"
+#include "src/common/system_logger.h"
+
+int main(int argc, char** argv) {
+  gfs::common::SystemLogger::GetInstance().Initialize(/*program_name=*/argv[0]);
+  const std::string config_filename("data/config.yml");
```

Comment:
it's easier to make these parsed from command line, so we can control (and potentially reuse this code) depending on which master and config we want to use/run the example against

Reply:
Yeah, I guess the thing here is that we only have a client library but not executable, so a client executable can do "anything" it wanted. This is just an example, but certainly has room for improvement.